### DISCUSSION TITLE
How to set the nozzle temperature to 300 degrees

### DISCUSSION THREAD

**Discussion by zkbutt** [mriscoc/Ender3V2S1#553] [Q&A]
I'm using this Ender3S1-F4-UBL-20221002  ，How to set the temperature to 300C or the hot bed to over 100C？

**Comment by mriscoc** [comment]
Stock hotend does not support 300°C, for Sprite extruder pro use the T13 special version. C104 custom G-code allows to unlock higher hotend temperatures. Other changes requiere that you modify the configuration files and compile your custom version.
